### ISSUE TITLE
Upgrade from 1.2.0Beta1 to 1.2.1, DB error

### ISSUE THREAD

**Issue by jackouille-ch** [Cacti/cacti#2368]
Hi,

I tried to migrate manually from 1.2.0Beta 1 to 1.2.1 on CentOS7 and got the following blocking error:
![image](https://user-images.githubusercontent.com/35721406/52162296-f7449c80-26d1-11e9-8fd5-52722e7d7ad5.png)

I was obliged to perform a rollback.

Here is my upgrade process:
cd /usr/share/
mv cacti old_cacti_1.2.0b1
cd /usr/share
cp -r /tmp/cacti-1.2.1/ cacti
cp  /usr/share/old_cacti_1.2.0b1/include/config.php /usr/share/cacti/include/config.php
cp -r old_cacti_1.2.0b1/rra/* cacti/rra
chmod -R 755 /usr/share/cacti/rra
cp -r old_cacti_1.2.0b1/scripts/* cacti/scripts
cp -r old_cacti_1.2.0b1/resource/* cacti/resource/
cp -r old_cacti_1.2.0b1/plugins/* cacti/plugins/
cp -r /tmp/cacti-1.2.1/scripts/* cacti/scripts
cp -r /tmp/cacti-1.2.1/resource/* cacti/resource/
cp -r /tmp/cacti-1.2.1/plugins/* cacti/plugins/
chown -R cacti:apache /usr/share/cacti/

What do I miss?

**Comment by jackouille-ch** [comment]
Thanks for your feedback

**Comment by jackouille-ch** [comment]
So after doing DB upgrade I was able to complete installer but but no graphs were present. I also rolled back once more.
My concern is that packages on CentOS 7 are delayed by several weeks or months so I used your beta release.
Now I will keep working with beta 1, waiting for the 1.2.1 official package. I have already installed a base VM and will then import my DB.
So I might be back with issues soon, or maybe and hopefully not.
